config/crd/bases/k6.io_k6s.yaml

```diff
@@ -857,31 +857,39 @@ spec:
                   - name
                   type: object
               type: object
-            scuttle:
-              properties:
-                enabled:
-                  type: string
-                envoyAdminApi:
-                  type: string
-                genericQuitEndpoint:
-                  type: string
-                istioQuitApi:
-                  type: string
-                neverKillIstio:
-                  type: boolean
-                neverKillIstioOnFailure:
-                  type: boolean
-                quitWithoutEnvoyTimeout:
-                  type: string
-                scuttleLogging:
-                  type: boolean
-                startWithoutEnvoy:
-                  type: boolean
-                waitForEnvoyTimeout:
-                  type: string
-              type: object
             separate:
               type: boolean
+            serviceMesh:
+              properties:
+                istio:
+                  properties:
+                    enabled:
+                      type: boolean
+                    envoyAdminApi:
+                      type: string
+                    genericQuitEndpoint:
+                      type: string
+                    istioQuitApi:
+                      type: string
+                    neverKillIstio:
+                      type: boolean
+                    neverKillIstioOnFailure:
+                      type: boolean
+                    quitWithoutEnvoyTimeout:
+                      type: string
+                    scuttleLogging:
+                      type: boolean
+                    startWithoutEnvoy:
+                      type: boolean
+                    waitForEnvoyTimeout:
+                      type: string
+                  type: object
+                linkerd:
+                  properties:
+                    enabled:
+                      type: boolean
+                  type: object
```

Comment:
Do you plan to add some other options here? I haven't read the docs on Linkerd yet but it looks quite small in comparison to all the Istio options :sweat_smile: